pkgs/development/python-modules/python-smarttub/default.nix

```diff
@@ -43,6 +43,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/mdz/python-smarttub";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ fab ];
-    broken = pyjwt.version != "1.7.1";
+    broken = pyjwt1.version != "1.7.1";
```

Comment:
I think we can drop that, since the 1.x series will probably not receive an update anyway.